src/ResourceManager/Sql/Commands.Sql/ServerDnsAlias/Cmdlet/RemoveAzureSqlServerDnsAlias.cs

```diff
@@ -0,0 +1,160 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Globalization;
+using System.Management.Automation;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+
+namespace Microsoft.Azure.Commands.Sql.ServerDnsAlias.Cmdlet
+{
+	[Cmdlet(VerbsCommon.Remove, "AzureRmSqlServerDnsAlias", SupportsShouldProcess = true)]
+	[OutputType(typeof(Model.AzureSqlServerDnsAliasModel))]
+	public class RemoveAzureSqlServerDNSAlias : AzureSqlServerDnsAliasCmdletBase
+	{
+		protected const string CmdletInputParameterSet =
+			"Remove a Server Dns Alias from cmdlet input parameters";
```

Comment:
@maboja-msft it looks like the parameter set names are still space-separate strings. I would recommend changing the names of them in this file to `RemoveByNameAndResourceGroup`, `RemoveByInputObject` and `RemoveByResourceId`